galaxy_importer/ansible_test/runners/local.py

```diff
@@ -16,10 +16,48 @@
 # along with Galaxy.  If not, see <http://www.apache.org/licenses/>.
 
 
+import os
+import subprocess
+
 from galaxy_importer.ansible_test.runners.base import BaseTestRunner
 
 
 class LocalAnsibleTestRunner(BaseTestRunner):
     """Run ansible-test locally with --docker or using venv."""
     def run(self):
-        pass
+        version_proc = subprocess.Popen(
+            ['ansible', '--version'],
+            stdout=subprocess.PIPE,
+            encoding='utf-8',
+        )
+        self.log.info(f'Using {list(version_proc.stdout)[0].rstrip()}')
+
+        suffix = f'ansible_collections/{self.metadata.namespace}/{self.metadata.name}/'
+        collection_dir = os.path.join(self.dir, suffix)
+
+        cmd = [
+            'ansible-test', 'sanity',
+            '--docker',
+            '--color', 'yes',
+            '--failure-ok',
+        ]
+
+        self.log.info(
+            'Running ansible-test sanity on '
+            f'{self.metadata.namespace}-{self.metadata.name}-{self.metadata.version} ...')
+        self.log.info(f'{" ".join(cmd)}')
+
+        proc = subprocess.Popen(
+            cmd,
+            cwd=collection_dir,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            encoding='utf-8',
+        )
+
+        for line in proc.stdout:
+            self.log.info(line.strip())
+
+        if proc.wait() != 0:
```

Comment:
Verify the Notes at https://docs.python.org/3.7/library/subprocess.html#subprocess.Popen.wait won't be a problem. Probably not, but since 'ansible test sanity' can be long lived and verbose it could creep up. But can also just leave it as is and worry about it later if it ever comes up.